tests/test_resource.py

```diff
@@ -338,15 +334,16 @@ def setup_method(self, _test_method, _init_mock: MagicMock) -> None:
         self.resource = PaginatedCORSResource({}, [])
         self.resource.path = "/test/path"
         self.resource.urn = "/test/path"
-        req.query_params = MultiDict(
+        self.req = pytest.mark.usefixtures("sample_request")
```

Comment:
This won't work as you expect. `pytest.mark.usefixtures` is a decorator, so you will get here the instance of one of `pytest` internal classes. Please. initialize the request normally 😉